gh-badges/lib/make-badge.js

```diff
@@ -164,18 +173,13 @@ function makeBadge({
     logoPosition,
     logoWidth,
     logoPadding,
-    // `color` and `labelColor` are included for the `default` JSON template.
-    color: normalizeColor(color),
-    labelColor: normalizeColor(labelColor),
     colorA: toSvgColor(labelColor),
     colorB: toSvgColor(color),
```

Comment:
`color` and `labelColor` are both now set to the return value of `normalizeColor`. Was it just an oversight before that `normalizeColor` wasn't used for `colorA` and `colorB`?

Reply:
For the purpose of the rendered badge, it doesn't matter whether or not you normalize. Capital or lowercase color codes are both recognized by the browser.

I guess it seems less surprising for the badge to use the same, normalized color, as the JSON response.

Reply:
> I guess it seems less surprising for the badge to use the same, normalized color, as the JSON response.

Completely agreed!